examples/websocket-retry-client/websocket_retry_client.bal

```diff
@@ -3,15 +3,11 @@ import ballerina/websocket;
 
 public function main() returns error? {
     websocket:Client wsClient = check new("ws://localhost:9090/foo", {
-        // Set the maximum retry count to 20 so that it will try 20 times with the default interval of
+        // Set the maximum retry count to 5 so that it will try 5 times with the interval of
         // 1 second in between the retry attempts.
-        retryConfig: { maxCount: 20 }
+        retryConfig: { maxCount: 5, interval: 1 }
```

Comment:
lets format this.

Reply:
Addressed with [a84428d](https://github.com/ballerina-platform/ballerina-distribution/pull/3559/commits/a84428dae7cfa580e8f1df7482a6c2851cede29e)